src/adapters/deno.ts

```
@@ -39,11 +42,21 @@ class DenoServer implements Server<DenoFetchHandler> {
 
     const fetchHandler = wrapFetch(this);
 
+    // Detect running in srvx loader
+    const loader = (globalThis as any).__srvxLoader__ as
+      | ((handler: ServerHandler) => void)
+      | undefined;
+    if (loader) {
+      this.fetch = fetchHandler;
+      loader(fetchHandler);
+      return this;
```

Biome (2.3.14)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)